src/Keboola/StorageApi/Client.php

```diff
@@ -1241,6 +1251,7 @@ public function dropToken($tokenId)
      *
      * @param string $tokenId If not set, defaults to self
      * @return string new token
+     * @deprecated $tokenId parameter will be removed in next major release. Use Tokens::refreshToken()
      */
     public function refreshToken($tokenId = null)
```

Comment:
Tohle jediné tady zůstane v příštím major releasu. Půjde ale přes to refreshnout jen aktuální token v clientovi.